### ISSUE TITLE
[Problem]: Restore from Backitup not working 

### ISSUE THREAD

**Issue by buanet** [buanet/ioBroker.docker#393]
### Description / Beschreibung

In my tests I had problems with restoring ioBroker from Backitup adapter.
Restoring from commandline (iobroker command or maintenance script) is working and a valid workaround. 

### Image version

v9.x.x

### Docker logs / Docker Protokoll

```shell
Restore attempt produces no log.
```

**Comment by buanet** [comment]
This is under investigation. If you have more information, please add it here. 

Regards,
André

**Comment by winnyschuster** [comment]
problem seems to be invocations of scripts in iobroker-data/backitup with `gosu iobroker`. see simple test from console:
```
root@iobtest:/opt/iobroker# su iobroker
iobroker@iobtest:~$ gosu iobroker ls
error: failed switching to "iobroker": operation not permitted
iobroker@iobtest:~$ 
```

when removing the gosu part, everything works as expected

**Comment by simatec** [comment]
What has changed in version 9.x...? Do I have to adjust anything in Backitup?

**Comment by winnyschuster** [comment]
looks like os behaviour has changed. above simple test is working fine under Debian11 but not with Debian12. I've also checked /etc/sudoers.d/iobroker files and they are the same except for paths, e.g `/bin/mount` vs. `/usr/bin/mnt`

**Comment by buanet** [comment]
Fixed with backitup [v2.8.7](https://github.com/simatec/ioBroker.backitup#287-2023-10-19).
